frontend/src/History/History.js

```diff
@@ -0,0 +1,93 @@
+import { useReduce } from "../Reducer-context";
+import TimeAgo from "react-timeago";
+import { getSortedData, getFilteredData } from "../App";
+import Navigator from "../Navigator";
+import { useEffect } from "react";
+import { Link } from "react-router-dom";
+
+import { Navbar } from "../Navbar";
+
+export default function History() {
+  useEffect(() => {
+    dispatch({ type: "CLEAR_FILTER" });
+  }, []);
+  const { history, dispatch, sortBy, showCategory } = useReduce();
+  console.log("history", history);
+
+  let sortedData = getSortedData(history, sortBy);
+  console.log("sorted", sortedData);
+  let filteredData = getFilteredData(sortedData, { showCategory });
+  console.log("filtered", filteredData);
+
+  function History() {
+    return filteredData
+      .filter((item) => item.ishistory !== false)
+      .map((item) => {
+        return (
+          <>
+            {" "}
```

Comment:
Add a name or indicator at the top of page if Videos are there to show the user he's in History videos